README.md

```diff
@@ -1,24 +1,27 @@
 # Homebrew
-Features, usage and installation instructions are [summarised on the homepage](http://brew.sh).
+**Features, usage and installation instructions are [summarised on the homepage](http://brew.sh).**
```

Comment:
I still believe we don't need the boldface in this line, too. It doesn't look as nice and that's not really something that needs to be emphasized …